hack/update-codegen.sh

```diff
@@ -44,6 +44,16 @@ ${KNATIVE_CODEGEN_PKG}/hack/generate-knative.sh "injection" \
   "samples:v1alpha1" \
   --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
 
+group "Update CRD Schema"
+
+go run $(dirname $0)/../cmd/schema/ dump SimpleDeployment \
+  | yq eval-all --header-preprocess=false --inplace 'select(fileIndex == 0).spec.versions[0].schema.openAPIV3Schema = select(fileIndex == 1) | select(fileIndex == 0)' \
+  $(dirname $0)/../config/300-simpledeployment.yaml -
+
+go run $(dirname $0)/../cmd/schema/ dump AddressableService \
+  | yq eval-all --header-preprocess=false --inplace 'select(fileIndex == 0).spec.versions[0].schema.openAPIV3Schema = select(fileIndex == 1) | select(fileIndex == 0)' \
```

Comment:
what's the select for?

Reply:
also `yq` isn't in our build image 

we do have a `run_go_tool` helper

you can see how to use it here

https://github.com/knative/serving/blob/80be89b59e2dd2ed5c351cec9d729fa1b4e450a3/test/e2e-common.sh#L568-L570

Reply:
it applies all the changes that were generated by running the `go run $(dirname $0)/../cmd/schema/ dump SimpleDeployment` (for example) in inside the `openAPIV3Schema` which is the data that is dumped from the schema dump

Reply:
Thanks for your review and feedback. I added the tool following the example you gave